package.json

```diff
@@ -47,13 +48,13 @@
     "node-fetch": "^2.2.0",
     "p-limit": "^2.0.0",
     "path-browserify": "^1.0.0",
-    "react": "16.5.0",
+    "react": "16.8.3",
     "react-intl": "^2.8.0",
     "react-markdown": "^4.0.6",
-    "react-native": "0.57.1",
+    "react-native": "0.59.3",
     "react-native-background-timer": "^2.1.1",
     "react-native-camera": "^1.3.0",
-    "react-native-cardano": "https://github.com/crossroadlabs/react-native-cardano.git",
+    "react-native-cardano": "https://github.com/Emurgo/react-native-cardano.git#upgrade-to-rn-059",
```

Comment:
I would try to merge that one before this one, so this PR can have the right path